comps/comps-foreman-plugins-rhel7.xml

```diff
@@ -136,30 +107,60 @@
       <packagereq type="default">tfm-rubygem-parse-cron</packagereq>
       <packagereq type="default">tfm-rubygem-pdf-core</packagereq>
       <packagereq type="default">tfm-rubygem-polyglot</packagereq>
       <packagereq type="default">tfm-rubygem-prawn</packagereq>
       <packagereq type="default">tfm-rubygem-puppetdb_foreman</packagereq>
+      <packagereq type="default">tfm-rubygem-radcli</packagereq>
       <packagereq type="default">tfm-rubygem-rainbow</packagereq>
       <packagereq type="default">tfm-rubygem-recursive-open-struct</packagereq>
+      <packagereq type="default">tfm-rubygem-route53</packagereq>
       <packagereq type="default">tfm-rubygem-rprogram</packagereq>
       <packagereq type="default">tfm-rubygem-rqrcode</packagereq>
+      <packagereq type="default">tfm-rubygem-ruby-hmac</packagereq>
       <packagereq type="default">tfm-rubygem-ruby-nmap</packagereq>
       <packagereq type="default">tfm-rubygem-safe_yaml</packagereq>
       <packagereq type="default">tfm-rubygem-sentry-raven</packagereq>
+      <packagereq type="default">tfm-rubygem-smart-proxy-probing</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_ansible</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_chef</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_dhcp_bluecat</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_dhcp_device42</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_dhcp_dnsmasq</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_dhcp_infoblox</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_dhcp_remote_isc</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_discovery</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_discovery_image</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_dns_infoblox</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_dns_powerdns</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_dns_route53</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_dynflow</packagereq>
       <packagereq type="default">tfm-rubygem-smart_proxy_dynflow_core</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_m2</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_monitoring</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_omaha</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_openscap</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_pulp</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_realm_ad_plugin</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_remote_execution_ssh</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_salt</packagereq>
```

Comment:
@evgeni Is all of this an expected reordering or do I need to change it to original?

Reply:
This looks perfectly reasonable. Seems we did not reorder the packages when we added the tfm prefix for the SCLed proxy.

Reply:
Great! For it to go into `2.1`, do I need to create a cherrypick PR yet?

Reply:
As far as I know 2.1 didn't branch yet.

Reply:
Awesome, thanks!